sample.py

```diff
@@ -1,2 +1,3 @@
 import os
 import math
```

Comment:
This is a Unusedimportmath
```suggestion 
```

 from our GitHub Enhacement Bot